### DISCUSSION TITLE
AIR33.1 Export release build creates .apk in bin-release-temp but not in designated export folder

### DISCUSSION THREAD

**Discussion by MacShrike** [airsdk/Adobe-Runtime-Support#2977] [Q&A]
Its all in the title really.
Added  <BuildLegacyAPK>true</BuildLegacyAPK> to remove the annoying no java home location for v11+ JDK.

After release build the only way to retrieve the .apk is by checking the keep bin-release-folder and then copying the .apk in that folder.

Any suggestions?

**Comment by ajwfrost** [comment]
To be honest, the best thing would be to move away from using Flash Builder... you could try VS Code or Moonshine IDE..?

The problem is that Flash Builder ships with a particular Java runtime and it's a pain to update it, but the old runtime isn't able to run the tools that are provided in the newer Android SDKs.

Plus there were other tweaks/workarounds that were needed to try to get past some of these things.. much better to move onto a supported toolchain, Adobe stopped supporting Flash Builder several years ago :-(

It may also be worth updating to a more recent AIR SDK if you're trying to publish onto the Play Store.

thanks
